pkg/common/model.go

```diff
@@ -101,3 +108,136 @@ type Annotation struct {
 	Key   string `json:"key"`
 	Value string `json:"value"`
 }
+
+type DerivedImageList struct {
+	Results []ImageSummary `json:"results"`
+}
+
+type BaseImageList struct {
+	Results [][]ImageSummary `json:"results"`
+}
+
+type FixedTags struct {
+	Errors []ErrorGQL `json:"errors"`
+	Data   struct {
```

Comment:
Let's find a solution which does not need us to type .Data. when we use this struct.